### PR TITLE
Remove declarations already declared by the new Windows headers

### PR DESCRIPTION
**Pull request by CyberShadow**
Requires https://github.com/D-Programming-Language/druntime/pull/1402

### CODE REVIEW

**Comment by CyberShadow** [comment]
No longer blocked, tests green.

File: std/file.d [hunk 1]
```diff
@@ -81,24 +81,6 @@ else version(Posix)
 }
 
 
-// @@@@ TEMPORARY - THIS SHOULD BE IN THE CORE @@@
```
Comment (Hackerpilot) [inline std/file.d]: I'm guessing that because the tests pass that the imports for these definitions were already present.

**Comment by Hackerpilot** [comment]
Auto-merge toggled on

File: std/file.d (hunk 1 above)
Comment (CyberShadow) [inline std/file.d]: They were added in https://github.com/D-Programming-Language/druntime/pull/1402

Comment (CyberShadow) [inline std/file.d]: Oh, the imports. Yeah.

**Pull request merged**